### PR TITLE
Enhance-get-all-interbank-transactions-with-pagination

### PR DESCRIPTION
**Pull request by nguyenthuthaochau2201**
## :sparkles: Summary

- Ticket: [Ticket link](https://trello.com/c/OzDCj3vx/44-admin-edit-endpoint-giao-d%E1%BB%8Bch-v%E1%BB%9Bi-m%E1%BB%99t-ng%C3%A2n-h%C3%A0ng)

## :ticket: Changes

1. Add query for get all interbank transactions with pagination
2. Add 2 new attributes to the response: soTienNhan & soTienGui

## :construction: Contributor checklist

-  [x] Run `pnpm build` and `pnpm start` successfully
-  [x] Checked that there are no additional warnings
-  [x] Added Github labels
-  [x] Code was formatted automatically

### CODE REVIEW

File: src/interbank/dto/query.dto.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export class QueryDTO {
```
Comment (phineasla) [inline src/interbank/dto/query.dto.ts:1]: **suggestion**: should be `InterbankTransactionQueryDto`

File: src/interbank/dto/query.dto.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  from: string;
+  to: string;
+  bankID: number;
```
Comment (phineasla) [inline src/interbank/dto/query.dto.ts:4]: **question**: does FE know bank ID? Should we change it to `bank: 'HCMUSBank'`?

File: src/interbank/interbank.service.ts [hunk 3]
```diff
@@ -73,13 +86,21 @@ export class InterbankService {
       throw e;
     }
     let interbankList;
+    let res = {};
```
Comment (phineasla) [inline src/interbank/interbank.service.ts:89]: **suggestion**: remove this

File: src/interbank/interbank.service.ts [hunk 4]
```diff
@@ -95,14 +116,52 @@ export class InterbankService {
       }
     }
     const lastPage = Math.ceil(total / pagination.size);
-
-    return formatResponse(
+    res = formatResponse(
```
Comment (phineasla) [inline src/interbank/interbank.service.ts:119]: **suggestion**: const res

Comment (nguyenthuthaochau2201) [reply src/interbank/interbank.service.ts:119]: I have to add two more attributes after that step. Can we new attributes to a const

Comment (phineasla) [reply src/interbank/interbank.service.ts:119]: Why not
https://stackoverflow.com/questions/23436437/why-can-i-change-a-constant-object-in-javascript

File: src/interbank/interbank.service.ts (hunk 3 above)
Comment (nguyenthuthaochau2201) [reply src/interbank/interbank.service.ts:89]: you mean ={} or the whole line

Comment (phineasla) [reply src/interbank/interbank.service.ts:89]: Use `const` here: https://github.com/phineasla/4.1-React-Karma-BE/pull/10#discussion_r1062126977

File: src/interbank/dto/query.dto.ts (hunk 2 above)
Comment (nguyenthuthaochau2201) [reply src/interbank/dto/query.dto.ts:4]: We will provide FE an api to get a list of interbanks so they will have bankID.

**Review by phineasla — APPROVED** [review]
<a href="https://imgflip.com/i/76kf6c"></a>

**Pull request merged**